Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -767,73 +824,266 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
+              const auto& itsOrig = recoData.getITSTrack(contributorsGID[GIndex::ITS]);
+              itsClusterMap = itsOrig.getPattern();
+            }
+            if (contributorsGID[GIndex::Source::TPC].isIndexSet()) {
+              isStoredTPC[track.getRefTPC()] = true;
+              const auto& tpcOrig = recoData.getTPCTrack(contributorsGID[GIndex::TPC]);
+              tpcChi2NCl = tpcOrig.getNClusters() ? tpcOrig.getChi2() / tpcOrig.getNClusters() : 0;
+              tpcSignal = tpcOrig.getdEdx().dEdxTotTPC;
+              tpcNClsFindable = tpcOrig.getNClusters();
+            }
+            // track
+            tracksCursor(0,
+                         collisionID,
+                         src,
+                         truncateFloatFraction(track.getX(), mTrackX),
+                         truncateFloatFraction(track.getAlpha(), mTrackAlpha),
+                         track.getY(),
+                         track.getZ(),
+                         truncateFloatFraction(track.getSnp(), mTrackSnp),
+                         truncateFloatFraction(track.getTgl(), mTrackTgl),
+                         truncateFloatFraction(track.getQ2Pt(), mTrack1Pt));
+            // track cov
+            tracksCovCursor(0,
+                            truncateFloatFraction(TMath::Sqrt(track.getSigmaY2()), mTrackCovDiag),
+                            truncateFloatFraction(TMath::Sqrt(track.getSigmaZ2()), mTrackCovDiag),
+                            truncateFloatFraction(TMath::Sqrt(track.getSigmaSnp2()), mTrackCovDiag),
+                            truncateFloatFraction(TMath::Sqrt(track.getSigmaTgl2()), mTrackCovDiag),
+                            truncateFloatFraction(TMath::Sqrt(track.getSigma1Pt2()), mTrackCovDiag),
+                            (Char_t)(128. * track.getSigmaZY() / track.getSigmaZ2() / track.getSigmaY2()),
+                            (Char_t)(128. * track.getSigmaSnpY() / track.getSigmaSnp2() / track.getSigmaY2()),
+                            (Char_t)(128. * track.getSigmaSnpZ() / track.getSigmaSnp2() / track.getSigmaZ2()),
+                            (Char_t)(128. * track.getSigmaTglY() / track.getSigmaTgl2() / track.getSigmaY2()),
+                            (Char_t)(128. * track.getSigmaTglZ() / track.getSigmaTgl2() / track.getSigmaZ2()),
+                            (Char_t)(128. * track.getSigmaTglSnp() / track.getSigmaTgl2() / track.getSigmaSnp2()),
+                            (Char_t)(128. * track.getSigma1PtY() / track.getSigma1Pt2() / track.getSigmaY2()),
+                            (Char_t)(128. * track.getSigma1PtZ() / track.getSigma1Pt2() / track.getSigmaZ2()),
+                            (Char_t)(128. * track.getSigma1PtSnp() / track.getSigma1Pt2() / track.getSigmaSnp2()),
+                            (Char_t)(128. * track.getSigma1PtTgl() / track.getSigma1Pt2() / track.getSigmaTgl2()));
+          }
+          if (src == GIndex::Source::ITSTPCTOF && mFillTracksITSTPC) {
```

Comment:
I think we should also store the label of the TOF cluster

Reply:
Hi @shahor02. Sorry, I don't quite understand: I think there is no column in the tracks tables in the data model (but I may be wrong)

Reply:
Yes, but I think it should be introduced, this is a MC label like others.
Btw, not there are also TPC-TRD and ITS-TPC-TRD tracks are added to the reconstruction (and RecoContainer), although at the moment w/o the MC labels.